src/types.jl

```diff
@@ -161,9 +143,8 @@ function Base.getproperty(font::FTFont, fieldname::Symbol)
     end
 end
 
-function Base.show(io::IO, font::FTFont)
+Base.show(io::IO, font::FTFont) =
```

Comment:
This is one is worth calling out, because one of my main deciding factors in when to use the one line form or  the `function` keyword is, whether the function definition actually fits into one line ;)
In other words, changes to the short function form are fine for functions that fit on a regular 80char line are fine, this one should definitely stay in the function form.

Reply:
Thanks, I've enforced the `80` width.